README.md

```diff
@@ -177,6 +189,9 @@ It is possible to specify a `releaseVersion` and `developVersion` on command lin
 grunt sg_release --releaseVersion 1.0.0 --developVersion 1.0.1-rc
 ```
 
+## Creating a hotfix
+A hotfix follows the same flow as a release but it usually branches off the master branch. All you have to do is to first checkout the master branch, then call the sg_release task. 
+
```

Comment:
What kind of example do you mean? You just checkout the master branch and then do the same as for a release.